src/utils/constants.ts

```diff
@@ -38,6 +38,8 @@ export const THEMES: Theme[] = [
       lightOpaque: "rgba(0, 0, 0, 0.05)",
       opaque: "rgba(0, 0, 0, 0.2)",
       opaqueActive: "rgba(0, 0, 0, 0.3)",
+      tertiaryBorderColor:"#9E9E9E",
+      tertiaryTextColor:"5C5C5C"
```

Comment:
`secondaryActive` and `primaryLightfont` should work here.. no need for these...

Reply:
done